packages/spiderable/spiderable_client_tests.js

```diff
@@ -0,0 +1,84 @@
+Tinytest.add("spiderable - default hooks registered", function (test, expect) {
+  test.equal(
+    _.keys(Spiderable._onReadyHook.callbacks).length,
+    2
+  );
+});
+
+Tinytest.add("spiderable - is not ready while initial subscriptions aren't started", function (test, expect) {
+  Spiderable._initialSubscriptionsStarted = false;
```

Comment:
I completely agree. A root issue is that `tinytest` provides zero utility for mocking/stubbing and in addition provides zero utility for setup and teardown (which would be really *really* nice). This was more so a shot in the dark and I wanted to at least give the `spiderable` *some* tests.